Readme installation

jrcain-usgs opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Reorganized the info in ISIS readme, moving some important info to ASC Docs, editing for conciseness, and removing unnecessary and overly verbose sections.

[View proposed new README](https://github.com/DOI-USGS/ISIS3/blob/739dbd93a91ecf9b573cc7a2205ad1c6dfe5f895/README.md)

## Description
<!--- Describe your changes in detail including motivation and any context -->
Extracted the Following Sections, moving each to its own ASC Doc page:

- Installing ISIS
- Setting Up the ISIS Data Area
- ISIS Test Data

Condensed sections remaining on README for now:

- RFC
- System Requirements
- Semantic Versioning

Updated section:

- Citing ISIS (with current 8.3.0 version DOI)

## Related Issue
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->
Fixes #5500 
Fixes #5650

## How Has This Been Validated?
<!--- All changes need to be validated to confirm that they produce -->
<!--- scientifically accurate results. -->
<!--- If your changes include any new algorithms or changes to existing algorithms, -->
<!--- please indicate any publications or references for them. -->
<!--- If you manually validated the changes please indicate -->
<!--- what data you used and how you checked the results. -->
Readme previewed via VS Code markdown preview rendering.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--- - [ ] My code follows the code style of this project. -->
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. <!--- NOTE: You can only have one changelog entry per PR, see https://github.com/DOI-USGS/ISIS3/blob/dev/CONTRIBUTING.md -->


## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

jrcain-usgs committed:
Update ISIS Readme, outsourced much of the content to ASC Docs

jrcain-usgs committed:
comments on TOC, quick reference

jrcain-usgs committed:
rfc2 versioning

Kelvinrr commented:
We can just change the logo like that, it would need to get reviewed by others first. I would argue to return it back to what it was and if we want to change it we can do something to update logos as a whole since some libraries might need ones.

jrcain-usgs committed:
logo

jrcain-usgs commented:
Changed logo back.

The reason to consider a different one is readability.

In the old logo, the ISIS text and the image behind are right on top of each other, and have a lot of intersecting lines between them, which limits readability.  

I changed it to brown for the ISIS docs to get better contrast, which helps but doesn't solve the root readability issue.

Kelvinrr commented:
@jrcain-usgs even if there are good reasons we need to have a conversation about these changes before they are worked, let alone merged. I would recommend also removing the logo from the PR.

Kelvinrr commented on README.md:
```diff
@@ -1,564 +1,126 @@
-is similar to the best practice usage of not adding any conda
-packages to your 'base' conda environment.
+- [Building ISIS](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/developing-isis3-with-cmake/)
+- [ISIS Test Data](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/obtaining-maintaining-submitting-test-data/)
```
We need something that is a more general "this is how you run ISIS tests" not just how to install isis test data. Those linked docs cover it but maybe name this link "'How to run ISIS tests"

jrcain-usgs replied:
"How to run ISIS Tests" is currently under the "Building ISIS" Doc.  [Here's a small PR to link to "ISIS Test Data" from "Building ISIS."](https://github.com/DOI-USGS/asc-public-docs/pull/127)  And with that, I can safely remove the ISIS Test Data link from the README.

Kelvinrr commented on README.md:
```diff
@@ -1,564 +1,126 @@
 
-The first step is to create 'working' and add whatever conda packages you want.
+- [Requests for Comment](README.md#Requests-for-Comment)
+- [Citing ISIS](README.md#Citing-ISIS)
```
These links are a little awkward since they link to the README in the repo (therefore loading a new page) rather than navigating to that section on the same page. iirc`(#Citing-ISIS)` should work?

Kelvinrr reviewed (commented):
just some comments

jrcain-usgs committed:
in-readme links, moved Test Data link into Building ISIS

jrcain-usgs committed:
remove isis-logo-red
